mgmtsystem/openstack.py

```diff
@@ -438,6 +438,7 @@ def deploy_template(self, template, *args, **kwargs):
         Args:
             template: The name of the template to use.
             flavour_name: The name of the flavour to use.
+            flavour_id: UUID of the flavour to use.
```

Comment:
Is there a specific reason not to do a lookup via neame and only specify the name, or visa versa?